### PR TITLE
APP-87: SDK Side

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @rasendubi 
/cc @Dubland

### CODE REVIEW

File: client/client-multi/client-c/toolchains/esp8266.cmake [hunk 1]
```diff
@@ -62,5 +62,5 @@ set(ESP8266_INCDIRS
     ${TOOLCHAIN_PATH}/lib/gcc/xtensa-lx106-elf/4.8.2/include/
     )
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-comment -fno-builtin -Wno-implicit-function-declaration -Os -Wpointer-arith  -Wl,-EL,--gc-sections -fno-inline-functions -nostdlib -mlongcalls -mtext-section-literals -ffunction-sections" CACHE FORCE "")
-set(CMAKE_EXE_LINK_EXECUTABLE "<CMAKE_C_COMPILER> <OBJECTS> <CMAKE_C_LINK_FLAGS> <LINK_FLAGS> -o <TARGET>")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-comment -fno-builtin -Wno-implicit-function-declaration -Os -Wpointer-arith -Wl,-EL,--gc-sections -fno-inline-functions -nostdlib -mlongcalls -mtext-section-literals -ffunction-sections" CACHE FORCE "")
+set(CMAKE_C_LINK_EXECUTABLE "<CMAKE_C_COMPILER> <FLAGS> <CMAKE_C_LINK_FLAGS> <LINK_FLAGS> -o <TARGET> -Wl,--start-group <OBJECTS> <LINK_LIBRARIES> -Wl,--end-group" CACHE FORCE "")
```
Comment (rasendubi) [inline client/client-multi/client-c/toolchains/esp8266.cmake:66]: `<FLAGS>` looks suspicious.

Comment (forGGe) [inline client/client-multi/client-c/toolchains/esp8266.cmake:66]: Seems that `<FLAGS>` are common placeholder for  `CMAKE_C_LINK_EXECUTABLE`. See [CMakeCInformation inside official Cmake repo](https://github.com/Kitware/CMake/blob/master/Modules/CMakeCInformation.cmake#L203)

**Comment by forGGe** [comment]
+

**Pull request merged**